src/main/java/com/mongodb/kafka/connect/sink/cdc/debezium/DebeziumCdcHandler.java

```diff
@@ -20,23 +20,16 @@
 
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Optional;
```

Comment:
The responsibility of respecting errors tolerance and error reporting is now all in one place.